AnnoDesigner/AnnoCanvas.xaml.cs

```diff
@@ -1522,21 +1576,30 @@ protected override void OnKeyDown(KeyEventArgs e)
         /// <summary>
         /// Checks whether the user is pressing the control key.
         /// </summary>
-        /// <returns></returns>
+        /// <returns><c>true</c> if the control key is pressed, otherwise <c>false</c>.</returns>
```

Comment:
We can use the (undocumented) `langword` attribute here. It can be used like this: `<see langword="true">`. That way we'll get nicely highlighted code in intellisense.

Reply:
I'm not familiar with this sort of documentation and VIsual Studio Code didn't give me any prompts. Can you take another look to make sure it is correct?